elastica/modules/operator_group.py

```diff
@@ -0,0 +1,62 @@
+from elastica.typing import OperatorType
+
+from collections.abc import Iterable
+
+import itertools
+
+
+class OperatorGroupFIFO(Iterable):
```

Comment:
FIFO does it mean first initialized first operated? Can we Write long maybe?

Reply:
FIFO: first in first out
I think it is common-enough terminology for queuing